### DISCUSSION TITLE
Periodic updates notifications via email

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#8428] [Feature Requests]
Following like Drupal style to notify updates/upgrades, security changes, why not add a section that will inform the user that there is a new version and the severity to take actions?

Right now, perhaps there is a way and I do not know it, but I only find out when is a new release because I follow the project and I receive the email of the new release from GitHub.

Sorry if it is repeated but I could not find a match...

**Comment by rijkvanzanten** [comment]
Interesting one. We do currently show a warning in the terminal, but I agree that might not be something that you necessarily see unless you look for it in a live deployment. 

I think this ties in nicely with https://github.com/directus/directus/discussions/7394. Once we have a notifications system in place that can be used to send/receive messages in the platform, we could utilize that to pull in messages from a centralized location to send messages to the open source installations (though we should make sure you can opt-out to those somehow)

**Reply by juancarlosjr97** [reply]
Yes, it looks like you are right and it will follow the notification systems idea, and I am glad it is in the pipeline :) 

Another thing I was thinking of is a status page to view the status and the compatibility of the system, something again like Drupal (sorry) [Drupal Status Page](https://www.drupal.org/docs/user_guide/en/prevent-status.html) that could potentially highlight if a new update made a version of the database outdated, the operating system, node version, etc. I know there is an information section on menu bar when visiting `/settings/project` but perhaps something like "ALERT you are using a very old PostgreSQL, upgrade now"

**Reply by rijkvanzanten** [reply]
Ahh yeah that's another good idea. I'm wondering where we'd pull that information from 🤔 It would mean that there has to be either a centralized location on the web this information can be pulled from, or we should keep a compatibility list within Directus itself, and check for versions on startup/runtime

**Reply by benhaynes** [reply]
Slightly related, I had this _old_ design for a Settings page to show your version compared to all available (presumably from the GitHub API or some master instance we manage).


![Settings-Server-Details](https://user-images.githubusercontent.com/522079/135318889-dec01c7d-a3cc-47e7-a817-649df274c4ca.png)

**Reply by juancarlosjr97** [reply]
Based on Drupal, the status checks and compatibility are local. 

```copied-from-drupal9-codebase
'title' => t('Database system version'),
      'value' => Database::getConnection()->version(),
    ];

    $errors = $tasks->engineVersionRequirementsCheck();
    $error_count = count($errors);
    if ($error_count > 0) {
      $error_message = [
        '#theme' => 'item_list',
        '#items' => $errors,
        // Use the comma-list style to display a single error without bullets.
        '#context' => ['list_style' => $error_count === 1 ? 'comma-list' : ''],
      ];
      $requirements['database_system_version']['severity'] = REQUIREMENT_ERROR;
      $requirements['database_system_version']['description'] = $error_message;
    }
```

A few that I think could be good would be SMTP and storage connection, and some other metrics/status...

The updates/upgrades of Drupal core are pulled from drupal along with any update/upgrade contribute modules that are compatible with the codebase, and it can be selected if notification is received on every update or only security updates.

![image](https://user-images.githubusercontent.com/36451129/135338992-f0ddc027-2596-48a6-b938-d68612931d5f.png)

However, all that information can be part of the status page, again like Drupal because it is quite good and simple

![localhost_4006_admin_reports_status](https://user-images.githubusercontent.com/36451129/135339078-d5229d6d-65b6-4f25-bdef-0931234e89c6.png)

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
